### DISCUSSION TITLE
Create index.tsx automatically with a task?

### DISCUSSION THREAD

**Discussion by Jarrodsz** [PhilDL/remix-gospel-stack#26] [Q&A]
Any suggestions on how to integrate the index export files automatically?

Its should be fairly trival but tried several index generators and they all s* 

My idea was to add a generate-index pnpm task so i can automate this and ser incorrect function declarations more easy while porting everything over to my mino repro setup

**Comment by PhilDL** [comment]
Hi, sorry I didn't really try these kinds of tools. 

In my opinion, when you are making a package, be it internal or bundled for npm, one goal is also to hide the internals. Then have a clean user-facing API with a few selected exports that you pick manually in your index.ts.

**Comment by Jarrodsz** [comment]
In the end I just define those files myself since the solutions out there did not work good.
Its basically a 1 time effort and then only needs updating better to manually do this process so you are sure your exports inside index.ts are correct. Thank you for the help Phil!
